src/main/java/nextstep/subway/path/dto/PathFinderResponse.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.subway.path.dto;
+
+import lombok.NoArgsConstructor;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
+import org.jgrapht.GraphPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@NoArgsConstructor
+public class PathFinderResponse {
+    private List<StationResponse> stations;
+    private double distance;
+
+    private PathFinderResponse(final GraphPath<Station, DefaultWeightedEdge> shortestPath) {
+        distance = shortestPath.getWeight();
+        stations = shortestPath.getVertexList()
+            .stream()
+            .map(StationResponse::of)
+            .collect(toList());
+    }
+
+    public static PathFinderResponse of(final GraphPath<Station, DefaultWeightedEdge> shortestPath) {
+        return new PathFinderResponse(shortestPath);
+    }
+
+    public List<StationResponse> getStations() {
+        return stations;
```

Comment:
크게 중요한 부분은 아니지만 `List`의 참조가 노출되니 아래와 같이 반환하면 어떨까요?

```suggestion
        return Collections.unmodifiableList(stations);
```

Reply:
아 dto 객체기 때문에 리뷰 주신대로 변경하는게 맞는 것 같습니다!!